deployment/templates/configmap-config.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.development }}
```

Comment:
I don't think there is another option here but this won't be fun to keep up to date.

Reply:
Added Note to deployment README regarding that this is a sample/subset and will not be kepy in sync with the official application.